x-pack/plugins/enterprise_search/server/lib/pipelines/update_default_pipeline.ts

```diff
@@ -35,8 +31,7 @@ export const updateDefaultPipeline = async (
       index: CURRENT_CONNECTORS_INDEX,
     });
   } catch (error) {
-    if (isIndexNotFoundException(error)) {
-      setupConnectorsIndices(client.asCurrentUser);
-    }
+    // TODO: Throw error saying you have to index a connector first
```

Comment:
Is it something we can do with this PR or should be done later ?

Reply:
Let's do it separately for now, I'll create an issue for it.